### PR TITLE
*/*: make it clear that <stabilize-allarches> ends in slash, />

### PR DESCRIPTION
**Pull request by juippis**
Modified all appearances of <stabilize-allarches>:
 - keywording/text.xml
 - ebuild-writing/misc-files/metadata/text.xml

### CODE REVIEW

**Comment by sbraz** [comment]
Wouldn't `pkgcheck` and `repoman` complain about that?

**Comment by juippis** [comment]
Yes, I thought it'd be nice to give a working example out-of-the-box for everyone. AFAIK this is the only tag that doesn't have content in metadata.xml.

File: keywording/text.xml [hunk 1]
```diff
@@ -348,7 +348,7 @@ arches that you would like to stabilize.
 
 <p>
 If your package is architecture-independent, you should add the
-<c>&lt;stabilize-allarches&gt;</c> tag to metadata.xml. This allows consistency
+<c>&lt;stabilize-allarches&gt;/</c> tag to metadata.xml. This allows consistency
```
Comment (ulm) [inline keywording/text.xml:351]: That's the only place where I'd add it.
However:
```suggestion
<c>&lt;stabilize-allarches/&gt;</c> tag to metadata.xml. This allows consistency
```

File: ebuild-writing/misc-files/metadata/text.xml [hunk 2]
```diff
@@ -120,7 +120,7 @@ metadata.xml:
   </ti>
 </tr>
 <tr>
-  <ti><c>&lt;stabilize-allarches&gt;</c></ti>
+  <ti><c>&lt;stabilize-allarches&gt;/</c></ti>
```
Comment (ulm) [inline ebuild-writing/misc-files/metadata/text.xml:123]: We don't mention a closing tag for any of the other tags, so I'd rather not for reasons of consistency.

Besides, the description says "This tag must have empty content."

File: keywording/text.xml (hunk 1 above)
Comment (juippis) [reply keywording/text.xml]: Heh, I guess I was blinded by the `</c>`. Re-did this based on your suggestion.

**Comment by ulm** [comment]
HTML afficionados would write `<stabilize-allarches />` (note the blank to appease the IE gods) or `<stabilize-allarches></stabilize-allarches>`.

Just kidding. :smile:

**Pull request closed without merging**